hr_employee_calendar_planning/migrations/13.0.1.3.0/post-migration.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    if version >= "13.0.1.0.0":
+        calendars_two_weeks = env["resource.calendar"].search(
+            [("auto_generate", "=", False), ("two_weeks_calendar", "=", True)]
+        )
+        calendars = env["resource.calendar"].search(
+            [("auto_generate", "=", False), ("two_weeks_calendar", "=", False)]
+        )
+        employees = env["hr.employee"].search(
+            [
+                ("calendar_ids.calendar_id", "in", calendars_two_weeks.ids),
```

Comment:
As these are subqueries, the AND conditions doesn't work properly. You have to loop on all employees.